en/contributing/build.md

```diff
@@ -264,16 +269,22 @@ docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000
 
 ### Building the Docker Image
 
-The approach above downloads a container image ([dronecore/dronecore](https://hub.docker.com/r/dronecore/dronecore/)) from Docker Hub.
+The approach above downloads a container image based on Ubuntu 16.04 ([dronecore/dronecore-ubuntu-16.04](https://hub.docker.com/r/dronecore/dronecore-ubuntu-16.04/)) or Fedora 27 ([dronecore/dronecore-fedora-27](https://hub.docker.com/r/dronecore/dronecore-fedora-27/)) from Docker Hub.
 
-You can also build the image yourself using the [Dockerfile](https://github.com/dronecore/DroneCore/blob/master/Dockerfile) in the root of the DroneCore repository (this is based on Ubuntu 16.04). The image can be used in the same way as the one from Docker Hub.
+You can also build the images yourself using the files in [DroneCore/docker](https://github.com/dronecore/DroneCore/tree/master/docker). The image can be used in the same way as the one from Docker Hub.
 
 1. Open a command prompt/terminal in the root of the DroneCore repository.
-1. Build the image as shown:
-   ```sh
-   docker build . -t my_image    # 'my_image' can then be used to refer to the image
-   ```
-1. Open a bash prompt using this image:
+1. Build the images as shown:
+   * Fedora 
+     ```sh
+     docker build -f docker/Dockerfile-Fedora-27 -t my_image .
+     ```
+   * Ubuntu 
+     ```sh
+     docker build -f docker/Dockerfile-Ubuntu-16.04 -t my_image .
```

Comment:
In my Ubuntu VM this Ubuntu dockerimage build fails with this gist: https://gist.github.com/hamishwillee/a41e2a76502b63f2926d3c6a698b13a9

Reply:
Ok, that's kind of expected. With upstream gRPC changes. Since we have our own gRPC fork anyway, we can remove this kind of stuff.

Reply:
Great, so this should be buildable "sometime soon"?

Reply:
See https://github.com/dronecore/DroneCore/pull/255